oblt-cli/deploy-my-kibana/action.yml

```
@@ -93,6 +95,8 @@ runs:
         else
           echo "CLUSTER=release-oblt" >> $GITHUB_ENV
```

We catch this with pre-commit in the obtl test environments repo, IIRC shellcheck

```suggestion
          echo "CLUSTER=release-oblt" >> "${GITHUB_ENV}"
```

---

We could change, however the official docs use the approach we use:
- https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#environment-files